src/components/views/rooms/SendMessageComposer.tsx

```diff
@@ -379,6 +443,7 @@ export class SendMessageComposer extends React.Component<ISendMessageComposerPro
                 }
 
                 if (cmd.category === CommandCategories.messages || cmd.category === CommandCategories.effects) {
+                    // TODO Do we need to attachMentions?
```

Comment:
Why wouldn't we? They're still text messages containing pills and the like

Reply:
I guess there's no reason not to; I think it was unclear to me if they could include mentions anywhere?

Reply:
You can use pills in e.g. `/fireworks <message>`